src/app.d.ts

```diff
@@ -8,6 +8,7 @@ declare global {
 		interface Platform {
 			env: {
 				DB: D1Database;
+				PRODUCTION: boolean;
```

Comment:
Is `PRODUCTION` a default Cloudflare binding that we don't have to explicitly set?

Reply:
noer (see above)